src/main/java/core/basesyntax/FileWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class FileWriterImpl implements FileWriter {
+
+    @Override
+    public void write(String report, String file) {
+        try {
+            Files.write(Path.of(file), report.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException("There was an error writing report.", e);
+        }
```

Comment:
It's recommended to use try-with-resources when dealing with IO operations to ensure that resources are closed properly even if an exception occurs. Consider wrapping the Files.write operation in a try-with-resources block.